Xamarin.Essentials/Launcher/Launcher.netstandard.watchos.cs

```diff
@@ -1,20 +0,0 @@
-﻿using System;
```

Comment:
@webwarrior-ws same here this PR should add GTK but not be destructive of other platforms